sql/structured.go

```diff
@@ -135,6 +135,11 @@ func (desc *TableDescriptor) SetName(name string) {
 	desc.Name = name
 }
 
+// IsSystem returns true if this is a system table.
+func (desc *TableDescriptor) IsSystem() bool {
+	return desc.GetID() == namespaceTableID || desc.GetID() == descriptorTableID
```

Comment:
yeah, I wasn't quite sure what we expected to use those for. Changed and added a comment in the ID section.